spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -387,6 +387,17 @@ func TestSample(t *testing.T) {
 	assertContains(t, out, "Updated data to VenueDetails column\n")
 	out = runSample(t, queryWithJsonParameter, dbName, "failed to query with json parameter")
 	assertContains(t, out, "The venue details for venue id 19")
+
+	out = runSample(t, createSequence, dbName, "failed to create table with bit reverse sequence enabled")
+	assertContains(t, out, "Created Seq sequence and Customers table, where the key column CustomerId uses the sequence as a default value\n")
+	assertContains(t, out, "Inserted customer record with CustomerId")
```

Comment:
Any way to check if this substring exists 3 times in out?

Reply:
Not needed as per offline discussion.